### PR TITLE
Added connections to final cave and sacred grounds

### PR DESCRIPTION
**Pull request by DodoBirby**

### CODE REVIEW

File: randovania/games/cave_story/json_data/Last Cave.txt [hunk 1]
```diff
@@ -4,26 +4,44 @@ Extra - map_name: Priso1
 Extra - in_dark_aether: False
 > Door to Plantation; Heals? False
   * Normal Door to Plantation/Door to Final Cave
+  > Door to Balcony (Pre-Bosses)
+      Trivial
 
 > Door to Balcony (Pre-Bosses); Heals? False
   * Normal Door to Balcony (Pre-Bosses)/Door to Final Cave
+  > Door to Plantation
+      Trivial
```
Comment (duncathan) [inline randovania/games/cave_story/json_data/Last Cave.txt]: these should probably have a Has Weapon check

File: randovania/games/cave_story/json_data/Last Cave.txt [hunk 2]
```diff
@@ -34,60 +52,90 @@ Extra - in_dark_aether: False
...
   * Normal Door to Last Cave (Hidden)/Door to Balcony (Pre-Bosses)
+  > Door to Prefab Building
+      Trivial
 
 > Door to Prefab Building; Heals? False
   * Normal Door to Prefab Building/Door to Balcony (Pre-Bosses)
+  > Exit to Throne Room
+      Trivial
+  > Door to Final Cave
+      Trivial
```
Comment (duncathan) [inline randovania/games/cave_story/json_data/Last Cave.txt]: this should be No Booster 2.0 (flip the ≥ to a < in the editor)

File: randovania/games/cave_story/json_data/Last Cave.txt [hunk 3]
```diff
@@ -98,19 +146,27 @@ Extra - in_dark_aether: False
 
 > Entrance from Throne Room; Heals? False
   * One-Way Entrance to Throne Room/Exit to Balcony (Post-Bosses)
+  > Event - Normal Ending
+      Trivial
+  > Exit to Prefab House
+      Trivial
```
Comment (duncathan) [inline randovania/games/cave_story/json_data/Last Cave.txt]: this requires Iron Bond, technically

File: randovania/games/cave_story/json_data/Last Cave.txt [hunk 4]
```diff
@@ -119,54 +175,88 @@ Extra - in_dark_aether: False
...
+      Booster 2.0
 
 ----------------
 Sacred Grounds - B2
 Extra - map_name: Hell2
 Extra - in_dark_aether: False
 > Door to Sacred Grounds - B1; Heals? False
   * Normal Door to Sacred Grounds - B1/Door to Sacred Grounds - B2
+  > Entrance to Sacred Grounds - B3
+      Trivial
```
Comment (duncathan) [inline randovania/games/cave_story/json_data/Last Cave.txt]: this DEFINITELY needs a weapon check

File: randovania/games/cave_story/json_data/Last Cave.txt [hunk 5]
```diff
@@ -177,6 +267,8 @@ Extra - map_name: Ballo1
 Extra - in_dark_aether: False
 > Entrance from Corridor; Heals? False
   * One-Way Entrance to Corridor/Exit to Seal Chamber
+  > Event - Best Ending
+      Can Kill Bosses
```
Comment (duncathan) [inline randovania/games/cave_story/json_data/Last Cave.txt]: this needs a Flight check, too

**Review by duncathan — CHANGES_REQUESTED** [review]

**Comment by DodoBirby** [comment]
Probably should have made these in seperate commits but... oh well

**Review by duncathan — APPROVED** [review]

**Pull request merged**